doc/smart_contract/tok_ctrt_no_split.md

```diff
@@ -0,0 +1,466 @@
+  functionIndex: 3,
+  functionData: '14uNyP1vcvYHknSVFL8C4hVccNhDi737TxjF58tBNvpEcAcNGf1',
+  attachment: 'YKYpT6ZcoWLMH2r6MBNvuoeLPFPCL8'
+}
+```
+
+#### Destroy
+
+Destroy a certain amount of the token.
+
+Note that only the address with the issuer role can take this action.
+
+```javascript
+// acnt: Accout from './src/account.js'
+// tc: TokCtrtWithoutSplit from './src/contract/tok_ctrt_no_split.js'
+
+let resp = await tc.destroy(
+  acnt, // by
+  50, // amount
+  'destroying some amount of token' // attachment
+);
+console.log(resp);
+```
+
+Example output
+
+```
+{
+  type: 9,
+  id: 'ECu4e6Rx5XRfjKFYVUdYBupb1SkaJXm2Nn9K3zhmDZh9',
+  fee: 30000000,
+  feeScale: 100,
+  timestamp: 1654659488012000000,
+  proofs: [
+    {
+      proofType: 'Curve25519',
+      publicKey: '6VH5QC2ktUA5UK4j6c4hxQTZi4cm9jdNYhnCQV2rT4Wv',
+      address: 'AU8xJNjE5RNo8hmPYA1bSgQzPKYNgejytiP',
+      signature: '3QKcT6b3kKiwX1FCztXbWLRR288rD9UTqvCf4CpRHUToUpqbFzJwGeVPjyQanKDf6tJk9uPQQF5YwdPQ9sPWCh3t'
+    }
+  ],
+  contractId: 'CEuFhD7N4sRfGnx6HZnCLnnd517Zgy19p4Q',
+  functionIndex: 2,
+  functionData: '14JDCrdo1xwuNP',
+  attachment: '2Xnstj5JdDLk4UoJ7mAAN3CGaaFmKRvUfzGzN5x16v9'
+}
+```
+
+#### Transfer
+
+Transfer a certain amount of the token to another account(e.g. user or contract).
+`transfer` is the underlying action of `send`, `deposit`, and `withdraw`. It is not recommended to use transfer directly. Use `send`, `deposit`, `withdraw` instead when possible.
+
+```javascript
+// acnt0: Accout from './src/account.js'
+// acnt1: Accout from './src/account.js'
+// tc: TokCtrtWithoutSplit from './src/contract/tok_ctrt_no_split.js'
+
+let resp = await tokB.transfer(
+  acnt0, // by
+  acnt0.addr.data, // sender
+  acnt1.addr.data, // recipient
+  10, // amount
+  'transfer to acnt1' // attachment
+);
+console.log(resp);
+```
+
+Example output
+
+```
+{
+  type: 9,
+  id: 'EErHX4BNCzpfqk7fbrSed2XLKth9qRrPta37XzvsjvKL',
+  fee: 30000000,
+  feeScale: 100,
+  timestamp: 1654659844659000000,
+  proofs: [
+    {
+      proofType: 'Curve25519',
+      publicKey: '6VH5QC2ktUA5UK4j6c4hxQTZi4cm9jdNYhnCQV2rT4Wv',
+      address: 'AU8xJNjE5RNo8hmPYA1bSgQzPKYNgejytiP',
+      signature: '26ZbVi2x8fMJrXfDf45ZsA2bcdS2iTacApeEwKwh4NaArgPTscEhZdL3vp4PLujkRMBmeXP84EYNSsn8qWffYeCY'
+    }
+  ],
+  contractId: 'CEuFhD7N4sRfGnx6HZnCLnnd517Zgy19p4Q',
+  functionIndex: 4,
+  functionData: '14VJY1ZthR99KumZMvcjjGnwmMggYiBsTzkczYTL1fh62FGgJmJfWBJ1yGDjL7nEHzYr6iGDzhVwK4KCQnjDxurf',
+  attachment: '26U7Yk1UsXAPPLDaRK4FPsoE'
+}
+```
+
+#### Deposit
+
+Deposit a certain amount of the token into a token-holding contract instance(e.g. lock contract).
+
+Note that only the token defined in the token-holding contract instance can be deposited into it.
+
+```javascript
+// acnt: Accout from './src/account.js'
+// tc: TokCtrtWithoutSplit from './src/contract/tok_ctrt_no_split.js'
+// lc: LockCtrt from './src/contract/lock_ctrt.js'
+
+lc_id = lc.ctrtId.data;
```

Comment:
```suggestion
lcId = lc.ctrtId.data;
```
parameters should follow camel case in javascript. Please check & update the rest of the doc as well.